docs/usage-guide/plugin-overview.md

```diff
@@ -0,0 +1,30 @@
+# Plugin Overview
```

Comment:
Can you add the Markdown header `order: N` such that we can control where in the user guide this page appears?